session/pingpong/invoice_payer.go

```diff
@@ -154,24 +159,7 @@ func (ip *InvoicePayer) Start() error {
 	}
 }
 
-const grandTotalKey = "consumer_grand_total"
-
-func (ip *InvoicePayer) getGrandTotalPromised() (uint64, error) {
```

Comment:
don't remove this func, the force-sync is temporary and we will need to operate on balances we have in state

Reply:
So we have `ip.deps.ConsumerTotalsStorage.Get()` for that and I created `recoverGrandTotalPromised()` for syncing it if needed